includes/admin/meta-boxes/views/html-related-orders-row.php

```diff
@@ -11,52 +11,52 @@
 ?>
 <tr>
 	<td>
-		<a href="<?php echo esc_url( get_edit_post_link( $order->get_id() ) ); ?>">
+		<a href="<?php echo esc_url( get_edit_post_link( $row_order->get_id() ) ); ?>">
 			<?php
 			// translators: placeholder is an order number.
-			echo sprintf( esc_html_x( '#%s', 'hash before order number', 'woocommerce-subscriptions' ), esc_html( $order->get_order_number() ) );
+			echo sprintf( esc_html_x( '#%s', 'hash before order number', 'woocommerce-subscriptions' ), esc_html( $row_order->get_order_number() ) );
 			?>
 		</a>
 	</td>
 	<td>
-		<?php echo esc_html( $order->get_meta( '_relationship' ) ); ?>
+		<?php echo esc_html( $row_order->get_meta( '_relationship' ) ); ?>
 	</td>
 	<td>
 		<?php
-		$date_created = $order->get_date_created();
+		$date_created = $row_order->get_date_created();
 
 		if ( $date_created ) {
-			$t_time          = $order->get_date_created()->date_i18n( get_option( 'date_format' ) . ' ' . get_option( 'time_format' ) );
+			$t_time          = $row_order->get_date_created()->date_i18n( get_option( 'date_format' ) . ' ' . get_option( 'time_format' ) );
 			$date_to_display = ucfirst( wcs_get_human_time_diff( $date_created->getTimestamp() ) );
 		} else {
 			$t_time = $date_to_display = __( 'Unpublished', 'woocommerce-subscriptions' );
 		}
 
 		// Backwards compatibility for third-parties using the generic WP post time filter.
-		$date_to_display = apply_filters( 'post_date_column_time', $date_to_display, get_post( $order->get_id() ) );
+		$date_to_display = apply_filters( 'post_date_column_time', $date_to_display, get_post( $row_order->get_id() ) );
```

Comment:
I think we should deprecate this filter, as it accepts a post parameter.

Reply:
Yep that sounds right to me.

Reply:
Actually, it'd be good to get your advice here. We don't own this filter (it's a WP filter). 

We could wrap this in an if statement, only fire this if HPOS disabled?

Reply:
👍 updated in 92fac53:

```php
		if ( ! wcs_is_custom_order_tables_usage_enabled() ) {
			// Backwards compatibility for third-parties using the generic WP post time filter.
			// Only apply this filter if HPOS is not enabled, as the filter is not compatible with HPOS.
			$date_to_display = apply_filters( 'post_date_column_time', $date_to_display, get_post( $row_order->get_id() ) );
		}
```